invenio_rdm_records/fixtures/tasks.py

```diff
@@ -27,3 +28,49 @@ def create_demo_record(data):
     service = current_rdm_records.records_service
     draft = service.create(data=data, identity=system_identity)
     service.publish(id_=draft.id, identity=system_identity)
+
+
+@shared_task(ignore_result=True)
+def update_expired_embargos():
+    """Release expired embargoes every midnight."""
+
+    def lift_embargo_from_draft(service, rdm_record):
+        """Lifts embargo from draft."""
+        record_draft = service.edit(rdm_record["id"], system_identity)
+        draft = record_draft._record
+        if draft.access.embargo.lift():
+            draft.access.protection.record = "public"
+            draft.access.protection.files = "public"
+        draft.commit()
+        return draft
+
+    def lift_embargo_from_record(rdm_record):
+        """Lifts embargo from record."""
+        if rdm_record.access.embargo.lift():
+            rdm_record.access.protection.record = "public"
+            rdm_record.access.protection.files = "public"
+            rdm_record.commit()
+
+    service = current_rdm_records.records_service
+    pid_type = "recid"
+    embargoed_q = "access.embargo.active:true"
+    restricted_records = service.scan(identity=system_identity, q=embargoed_q)
+    for restricted_record in restricted_records.to_dict()["hits"]["hits"]:
+        record = service.resolve_pid(
```

Comment:
The idea behind having a `result_item` is that there *should not* be any difference when a result comes from DB or ES. Therefore, this call *should* be redundant and the object you are getting *should* be the same than `restricted_record`. Note all the *should*, meaning that if it's not like that you might have found a bug ;)

Reply:
Hmm, I am not sure this is a bug but the call is needed since `restricted_records`  is a `RecordList` and we get the hit (`restricted_record`) that is only a `dict`. In the other hand when we resolve the pid we do get a `RecordItem`.

Reply:
This is because of the `.to_dict()["hits"]["hits"]:`, you could do:
```python
for restricted_record in restricted_records.hits:
```